django/utils/http.py

```diff
@@ -293,12 +295,21 @@ def is_safe_url(url, host=None):
             url = force_text(url)
         except UnicodeDecodeError:
             return False
+    if allowed_hosts is None:
+        allowed_hosts = set()
+    if host:
+        warnings.warn(
```

Comment:
I get your point but as `host` is only kept for backward compatibility reasons on an undocumented API I don't think we should refrain from using a more appropriate name for the sake of consistency with it. Anyway, we're stepping into bikeshedding territory here, both options seem viable to me.

@timgraham, @jdufresne thoughts?